howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+            direct_message_receiver(DirectMessageReceiver): an already configured and connected receiver
+
+        Raises:
+            InterruptedException: if any thread has interrupted the current thread
+            PubSubPlusClientException: if the operation could not be performed, i.e. when
+            cache request failed and subscription rolled back/removed
+            IllegalStateException: If the service is not running
+        """
+        cached_message_subscription_request = CachedMessageSubscriptionRequest.asAvailable(cache_name,
+                                                                                           TopicSubscription.of(subscription_expression),
+                                                                                           cache_access_timeout)
+        correlation_id = 12345
+        correlation_completion_listener = MyCorrelationCompletionListener()
+
+        direct_message_receiver.requestCached(cached_message_subscription_request, correlation_id, correlation_completion_listener)
+
+    def create_subscription_request_to_receive_latest_message(direct_message_receiver: 'DirectMessageReceiver',
+                                                              subscription_expression: str,
+                                                              cache_name: str,
+                                                              cache_access_timeout: int):
+        """
+        An example of how to create and perform CachedMessageSubscriptionRequest to receive latest messages. When no live
+        messages are available, cached messages matching specified topic subscription considered latest, live messages otherwise.
+        When live messages are available then cached messages are discarded.
+
+        Args:
+            direct_message_receiver(DirectMessageReceiver): An already configured and connected receiver.
+            subscription_expression(str): Topic subscription expression.
+            cache_name(str): name of the solace cache to retrieve from
+            cache_access_timeout(int): Solace cache request timeout, in milliseconds
+
+        Raises:
+            InterruptedException: If any thread has interrupted the current thread
+            PubSubPlusClientException: If the operation could not be performed
+            IllegalStateException: If the service is not running
+        """
+        cached_message_subscription_request_for_latest_of_cached_or_live_messages = \
+            CachedMessageSubscriptionRequest.live_cancels_cached(cache_name,
+                                                                 TopicSubscription.of(subscriptionExpression),
+                                                                 cache_access_timeout)
+
+        correlation_id = 12345
+        correlation_completion_listener = MyCorrelationCompletionListener()
+
+        direct_receiver.request_cached(cached_message_subscription_request_for_latest_of_cached_or_live_messages,
```

Comment:
Parameter is named `direct_message_receiver` not `direct_receiver`

Reply:
Please review fix